src/main/java/com/conveyal/gtfs/graphql/fetchers/JDBCFetcher.java

```diff
@@ -42,12 +45,19 @@
     public static final String LIMIT_ARG = "limit";
     public static final String OFFSET_ARG = "offset";
     public static final String SEARCH_ARG = "search";
+    public static final String DATE_ARG = "date";
```

Comment:
Is it really necessary to define symbolic constants that have exactly the same name as their own contents? I guess the advantage is that if they're reused in several places, there's no risk of a misspelling in one place, since they're checked by the IDE and compiler.

Reply:
Yes, reusability was my reasoning here. If the arguments need to be added to other types, it's also much more convenient and less error prone to type out a constant than a string literal in the IDE.